docs/user-guide/troubleshooting.rst

```diff
@@ -97,7 +97,7 @@ in ``pulp-admin tasks`` commands.  An example of a log statement with a task id
 Here the shortened task id is ``338bdde4``. The full task id would be
 ``338bdde4-608a-44ab-a79c-49c28b0fe037``. A statement without a task id would look like:
```

Comment:
You didn't introduce this, but when I rendered these docs the indendented portions here aren't blockquoted. It's because the `:` needs to become a `::` instead. If you render the docs locally (`make clean;make html`) you'll see what I mean. The same problem exists also on L#93.